<launch>
    <arg name="ACM_name"/>
    <node pkg="dji_osdk_ros" type="dji_vehicle_node" name="vehicle_node" output="screen">
    <!-- node parameters -->
    <param name="acm_name" type="string" value="$(arg ACM_name)"/>
    <param name="serial_name" type="string" value="/dev/ttyTHS1"/>
    <param name="baud_rate" type="int" value="1000000"/>
    <param name="app_id" type="int" value="1098852"/>
    <param name="app_version" type="int" value="1"/>
    <param name="align_time" type="bool" value="false"/>
    <param name="enc_key" type="string" value="e3b237bee1bcfb56e81f8c483944f74ba1e0247654880e2a3580e401893bd1ea"/>
    <param name="use_broadcast" type="bool" value="false"/>
    </node>
</launch>
